### PR TITLE
Include akkaHttpVersion in PlayVersion

### PR DESCRIPTION
**Pull request by marko-asplund**
# Pull Request Checklist

* [x] Have you read [How to write the perfect pull request](https://github.com/blog/1943-how-to-write-the-perfect-pull-request)?
* [x] Have you read through the [contributor guidelines](https://www.playframework.com/contributing)?
* [x] Have you signed the [Lightbend CLA](https://www.lightbend.com/contribute/cla)?
* [x] Have you referenced any issues you're fixing using [commit message keywords](https://help.github.com/articles/closing-issues-using-keywords/)?
* [x] Have you added copyright headers to new files?
* [x] Have you checked that both Scala and Java APIs are updated?
* [x] Have you updated the documentation for both Scala and Java sections?
* [x] Have you added tests for any changed functionality?

# Helpful things

## Fixes

N/A

## Purpose

Makes `akkaHttpVersion` available through `PlayVersion`.

## Background Context

N/A

## References

N/A

### CODE REVIEW

**Comment by ignasi35** [comment]
context: https://gitter.im/playframework/playframework?at=5efa3803a813c72dcf19421d

File: project/Tasks.scala [hunk 1]
```diff
@@ -24,6 +25,7 @@ object Generators {
           |  val scalaVersion = "$scalaVersion"
           |  val sbtVersion = "$sbtVersion"
           |  val akkaVersion = "$akkaVersion"
+          |  val akkaHttpVersion = "$akkaHttpVersion"
```
Comment (ignasi35) [inline project/Tasks.scala:28]: If adding this requires the dependencies above maybe we need to rethink this generator.

but I think the dependency should not be necessary. I mean, below this line the generator adds `val jettyAlpnAgentVersion` which is not always available.

File: build.sbt [hunk 2]
```diff
@@ -84,6 +84,7 @@ lazy val PlayProject = PlayCrossBuiltProject("Play", "core/play")
           sbtVersion.value,
           jettyAlpnAgent.revision,
           Dependencies.akkaVersion,
+          Dependencies.akkaHttpVersion,
           (sourceManaged in Compile).value
         )
```
[the remark this answers is not in the document]
Comment (marko-asplund) [reply build.sbt:89]: @ignasi35 😕  Sorry, but I'm not sure I understand - could you please elaborate.

On lines 87 and 236 in `build.sbt` we're referencing an eagerly evaluated `String` (`Dependencies.akkaHttpVersion`). The previous line in both call sites reference `Dependencies.akkaVersion`  which should result in `Depencencies` getting evaluated in both cases. `jettyAlpnAgent` OTOH is of type `ModuleID`.

[the remark this answers is not in the document]
Comment (ignasi35) [reply build.sbt:89]: 🤦🏼‍♂️ You're 100% right. 

LGTM!

[the remark this answers is not in the document]
Comment (marko-asplund) [reply build.sbt:89]: Thanks @ignasi35 & @raboof ! 🙇

**Pull request merged**